docs/eips/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ethereum Improvement Proposals (EIPs)
+meta:
+  - name: description
+    content: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+  - property: og:title
+    content: Ethereum Improvement Proposals (EIPs) | Ethereum.org
+  - property: og:description
+    content: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+lang: en-US
+sidebar: auto
+---
+
+# Introduction to Ethereum Improvement Proposals (EIPs)
+
+## What is an EIP?
+
+[Ethereum Improvement Proposals (EIPs)](https://eips.ethereum.org/) are standards specifying potential new features or processes for Ethereum. EIPs contain technical specifications for the proposed changes and act as the “source of truth” for the community. Network upgrades for Ethereum are discussed and developed through the EIP process.
+
+EIP authors are commonly core developers but anyone within the Ethereum community has the ability to create an EIP. Guidelines for writing EIPs are included in [EIP 1](https://eips.ethereum.org/EIPS/eip-1). The EIP should provide a concise technical specification of the feature its rationale. The EIP author is responsible for building consensus within the community and documenting dissenting opinions.
+
+## Why do EIPs matter?
+
+EIPs play a central role in how changes happen and are documented on Ethereum. They are the way for people to propose, debate and adopt changes. There are [different types of EIPs](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-types) including core EIPs for low-level protocol changes that affect consensus and require a network upgrade and ERCs for community standards. For example, standards to create tokens, like [ERC20](https://eips.ethereum.org/EIPS/eip-20) or [ERC721](https://eips.ethereum.org/EIPS/eip-721) allow applications interacting with these tokens to all treat tokens using the same rules, which makes it easier to create interoperable applications.
```

Comment:
Should it be "community standards" or "application standards" 🤔 ?

Reply:
I'm going with "application standards" for now. I'm not sure what an example of a "community standard" would be.